scripts/upload_artifact_s3.py

```diff
@@ -0,0 +1,38 @@
+import os
+import boto3
+import datetime
+from botocore.exceptions import ClientError
+
+path = "."
+extension = ".tar.gz"
+
+
+def upload_artifact_s3(aws_env):
+    """Upload artifact to the specified S3 bucket.
+    Specification done by environment variable AWS_BUCKET.
+    returns:
+        True if upload successful.
+        False otherwise.
+    """
+
+    s3_client = boto3.client(service_name="sqs",
+                             region_name=aws_env["AWS_REGION"],
+                             aws_access_key_id=aws_env["AWS_ACCESS_KEY_ID"],
+                             aws_secret_access_key=aws_env["AWS_SECRET_ACCESS_KEY"])
+
+    for file in os.listdir(path):
+        if file.endswith(extension):
+            file_name = file
```

Comment:
so it will upload just one file. is it expected?

Reply:
The file is a .tar.gz so yes